site/content/kapp/docs/develop/apply.md

```diff
@@ -67,6 +67,16 @@ Possible values: "" (default).
 
 Annotation value will be replaced with a unique ID on each deploy. This allows to force resource update as value changes every time.
 
+### kapp.k14s.io/renew-duration
+
+Available in v0.54.0+
+
+`kapp.k14s.io/renew-duration` annotation allows setting an interval after which you want your resource to get updated/renewed.
+
+Possible values: [ParseDuration](https://pkg.go.dev/time#ParseDuration).
+
+**Note**: For precise results use the `versioned resources` or `always-replace` update strategy for `non-versioned resources`.
```

Comment:
I think we should be more specific about cons here and what "precise results" mean